src/components/menu-bar/account-nav.jsx

```diff
@@ -45,57 +46,46 @@ const AccountNav = ({
                 {username}
             </span>
         </a>
-        <Dropdown
-            as="ul"
-            className={process.env.SCRATCH_ENV}
-            isOpen={accountNavOpen}
+        <MenuBarMenu
+            open={accountNavOpen}
+            // note: the Rtl styles are switched here, because this menu is justified
+            // opposite all the others
+            place={isRtl ? 'right' : 'left'}
             onRequestClose={onCloseAccountNav}
         >
-            <li>
-                <a href={profileUrl}>
-                    <FormattedMessage id="general.profile" />
-                </a>
-            </li>
-            <li>
-                <a href="/mystuff/">
-                    <FormattedMessage id="general.myStuff" />
-                </a>
-            </li>
-            {isEducator ? [
-                <li key="my-classes-li">
-                    <a href="/educators/classes/">
-                        <FormattedMessage id="general.myClasses" />
-                    </a>
-                </li>
-            ] : []}
-            {isStudent ? [
-                <li key="my-class-li">
-                    <a href={`/classes/${classroomId}/`}>
-                        <FormattedMessage id="general.myClass" />
-                    </a>
-                </li>
-            ] : []}
-            <li>
-                <a href="/accounts/settings/">
-                    <FormattedMessage id="general.accountSettings" />
-                </a>
-            </li>
-            <li className={dropdownStyles.divider}>
-                <a
-                    href="#"
-                    onClick={onLogOut}
-                >
+            <MenuItem href={profileUrl}>
+                <FormattedMessage id="general.profile" />
+            </MenuItem>
+            <MenuItem href="/mystuff/">
+                <FormattedMessage id="general.myStuff" />
+            </MenuItem>
+            {isEducator ? (
+                <MenuItem href="/educators/classes/">
+                    <FormattedMessage id="general.myClasses" />
+                </MenuItem>
+            ) : []}
```

Comment:
I think `[]` works just as well, but the convention is `null`.

Reply:
Done